packages/esm-patient-search-app/src/patient-search-result/result-card/result-card.component.tsx

```diff
@@ -0,0 +1,127 @@
+import { age, ExtensionSlot, formatDate, navigate, parseDate, useVisit } from '@openmrs/esm-framework';
+import { Button, ClickableTile } from 'carbon-components-react';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './result-card.scss';
+import ChevronDown16 from '@carbon/icons-react/es/chevron--down/16';
+import ChevronUp16 from '@carbon/icons-react/es/chevron--up/16';
+import OverflowMenuVertical16 from '@carbon/icons-react/es/overflow-menu--vertical/16';
+import CustomOverflowMenuComponent from './overflow-menu.component';
+import ContactDetails from '../contact-details/contact-details.component';
+import { capitalize } from 'lodash-es';
+
+interface ResultCardProp {
+  patient: fhir.Patient;
+  onSearchResultClick: (patientUuid: string) => void;
+  closeSearchResultsPanel: () => {};
+}
+
+const ResultCard: React.FC<ResultCardProp> = ({ patient, onSearchResultClick, closeSearchResultsPanel }) => {
+  const { t } = useTranslation();
+  const overFlowMenuRef = React.useRef(null);
+  const { currentVisit } = useVisit(patient.id);
+  const shouldStartVisit = (patient.deceasedDateTime ? false : true) && currentVisit === null;
+  const [showContactDetails, setShowContactDetails] = useState<boolean>(false);
+  const patientName = `${patient.name?.[0].given?.join(' ')} ${patient?.name?.[0].family}`;
+  const photoFrameState = useMemo(() => ({ patientUuid: patient.id, patientName }), [patient, patientName]);
+
+  const patientGender = () => {
+    if (patient.gender === 'M') {
+      return t('male', 'Male');
+    }
+    if (patient.gender === 'F') {
+      return t('female', 'Female');
+    }
+    return t('unknown', 'UnKnown');
+  };
```

Comment:
We should have a `getDisplayGender(genderCode)` function in esm-styleguide or esm-utils so that we're doing it the same way everywhere (and the translation lives in exactly one place). Not needed for this PR to go in though.

Reply:
CC @ZacButko @denniskigen  , thoughts?